operator/operator-example-standalone.yaml

```
@@ -0,0 +1,166 @@
+
+      helmChartDependencies:
+        - chartName: redis-operator
+          chartVersion: 0.19.2
+          chartRepoName: ot-helm
+          chartRepoURL: "https://ot-container-kit.github.io/helm-charts/"
+          chartValues:
+            redisOperator:
+              name: redis-operator
+              imageName: muhammadqadora/operator-hostport
+              # Overrides the image tag whose default is the chart appVersion.
+              imageTag: "3.2"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)